conftest.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import os
+import subprocess
+
+import pytest
+
+# Dirty hack for adding the module discoverable (for now)
+import sys
+
+sys.path.append(os.getcwd() + "/tests")
```

Comment:
Same as [here](https://github.com/mendersoftware/mender-convert/pull/262#discussion_r531133156) and [here](https://github.com/mendersoftware/meta-mender/pull/1191/files#r531137186) :)